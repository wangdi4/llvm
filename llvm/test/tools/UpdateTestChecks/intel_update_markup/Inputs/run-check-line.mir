# RUN: llc -O2 < %s | FileCheck --check-prefixes=COMMON,O2 %s
# RUN: llc -O3 < %s | FileCheck %s
--- |
; RUN: llc -O0 < %s | FileCheck --check-prefixes=COMMON,O0 %s
...
---
# COMMON: a
# O0: b
# O2: c
# CHECK: d
# O0 comment
## O2 comment
MIR
