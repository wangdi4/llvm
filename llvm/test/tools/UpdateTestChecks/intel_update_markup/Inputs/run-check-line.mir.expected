# RUN: llc -O2 < %s | FileCheck --check-prefixes=COMMON,O2 %s ;INTEL
# RUN: llc -O3 < %s | FileCheck %s ;INTEL
--- |
; RUN: llc -O0 < %s | FileCheck --check-prefixes=COMMON,O0 %s ;INTEL
...
---
# COMMON: a ;INTEL
# O0: b ;INTEL
# O2: c ;INTEL
# CHECK: d ;INTEL
# O0 comment # INTEL
## O2 comment # INTEL
MIR # INTEL
